styles/alert/alert.css

```diff
@@ -0,0 +1,51 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
+
+.alert {
+  padding: 1rem;
+  margin: 1rem;
+  font-weight: bold;
+}
+
+.alert-primary {
+  color: rgb(11, 230, 230);
+  background-color: var(--default);
+}
+.alert-secondary {
+  color: rgb(165, 103, 11);
+  background-color: rgb(245, 245, 185);
```

Comment:
define these colors in root first and then use it

Reply:
done